hax/hax.ini

```diff
@@ -144,6 +144,9 @@ corrections_definitions = {
         {"run_min": 0, "correction": "s1-lce_kr83m-sr0_pax-680_fdc-3d_v0.json"},
         {"run_min": 6386, "correction": "s1-lce_kr83m-sr1_pax-680_fdc-3d_v0.json"}
     ],
+    "s1_true_lce_map_nn_fdc_3d": [
+        {"run_min": 6386, "correction": "XENON1T_s1_xyz_lce_true_kr83m_SR1_pax-680_fdc-3d_v0.json"}
```

Comment:
Is this map using the last iteration? Should it instead be the average of the last few iterations, since it's oscillating?

Reply:
Hi Pats,
thanks a lot for making the commit, I wasn't around a pc last days. You are right, the map generated currently comes from the last iteration, since for the inner part of the TPC it seems to be sort of a close to the value of convergance. I could also f¿define after these iterations one extra stage just taking the average value from each FV and generate the map. Would you prefer that?

Reply:
yes, @mcfatelin agree?

(i didn't make any commit?)

Reply:
Sorry I missed this thread. Actually I agree with Patrick that we shall use the average electron lifetime from the last several iterations for extended FV in SR1.

Reply:
Ciao @pdeperio 
I just wanted to tell you that I just finished the new LCE corrected map introducing your suggestion, this is taking the average (convergence value) for the electron lifetimes in order to define a final Stage 10 correction map. Here are the results:

![stages_iteration_calibration_3_feb](https://user-images.githubusercontent.com/16705375/34586521-9b5f7e34-f1a4-11e7-83d5-38cd4de411b4.png)

This would be the map (for SR1):

[LCE_True_kr83m_stage_10_3d_FDC_t_dep_Calibration_3_Feb_pax-680.json.zip](https://github.com/XENON1T/hax/files/1604876/LCE_True_kr83m_stage_10_3d_FDC_t_dep_Calibration_3_Feb_pax-680.json.zip)

which lead to similar comparison results as before:
![corrections_comparison_calibration_3_feb](https://user-images.githubusercontent.com/16705375/34586659-2e4edf82-f1a5-11e7-82a5-f6f9c01800da.png)

P.S: Since I'm assuming that we may need too the map for SR0, I'm working already on generating it before making the PR. I'll have it tomorrow morning since this one has to be calculated all again! Let me know if it is what you had in mind and if you have any other suggestion. until tomorrow with the PR.

Reply:
This looks goods! Thank you!

Reply:
Here are the results up to Stage 6 for SR0 to generate the LCE corrected map just as done for SR1, I'll now proceed to make the PR:

![stages_iteration_sr0](https://user-images.githubusercontent.com/16705375/34603739-8178f232-f205-11e7-9858-2c87a6dc79c9.png)

This would be the map (SR0):

[LCE_True_kr83m_stage_6_3d_FDC_t_dep_SR0_pax-680.json.zip](https://github.com/XENON1T/hax/files/1606141/LCE_True_kr83m_stage_6_3d_FDC_t_dep_SR0_pax-680.json.zip)

which lead to the following comparison of results:

![corrections_comparison_sr0](https://user-images.githubusercontent.com/16705375/34603806-af11e924-f205-11e7-8a50-ac82ce98697d.png)